lib/pos_blockchain.go

```diff
@@ -368,20 +366,30 @@ func (bc *Blockchain) validateQC(desoBlock *MsgDeSoBlock, validatorSet []*Valida
 	return nil
 }
 
-// validateTimeoutQC validates that the parent block hash is not too far back from the latest.
-// Specifically, it checks that the parent block hash is at least the latest committed block.
-func (bc *Blockchain) validateTimeoutQC(desoBlock *MsgDeSoBlock, validatorSet []*ValidatorEntry) error {
-	// TODO: Implement me
-	return errors.New("IMPLEMENT ME")
-}
-
-// hasKnownAncestors checks that the parent block hash exists in the block index.
-// It returns true if the parent block hash exists in the block index, and otherwise false.
-// If the parent hash is in the block index, it is assumed that all ancestors of this block
-// also exist in the block index.
-func (bc *Blockchain) hasKnownAncestors(parentHash BlockHash) bool {
-	_, exists := bc.blockIndex[parentHash]
-	return exists
+// getAncestorsToCommittedTip returns the ancestors of the block provided up to the
+// committed tip.
+func (bc *Blockchain) getAncestorsToCommittedTip(desoBlock *MsgDeSoBlock) ([]*BlockNode, error) {
```

Comment:
Wdyt of this name as an alternative? The function returns the sequence of blocks beginning from the committed tip up until the input block (exclusive of both)
```suggestion
func (bc *Blockchain) getLineageFromCommittedTip(desoBlock *MsgDeSoBlock) ([]*BlockNode, error) {
```